addons/sale_stock/models/res_users.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class Users(models.Model):
+    _inherit = ['res.users']
+
+    LOW_ACCESS_RIGHTS_GROUPS = 'stock.group_stock_user,sales_team.group_sale_salesman,base.group_erp_manager'
+    HIGH_ACCESS_RIGHTS_GROUPS = 'stock.group_stock_manager,sales_team.group_sale_manager,base.group_erp_manager'
+
+    can_read_warehouse = fields.Boolean(compute='_compute_can_read_warehouse')
+    can_edit_warehouse = fields.Boolean(compute='_compute_can_edit_warehouse')
+    property_warehouse_id = fields.Many2one('stock.warehouse', string='Default Warehouse', company_dependent=True, check_company=True)
+
+    def _compute_can_read_warehouse(self):
+        has_low_access_rights_groups = self.user_has_groups(self.LOW_ACCESS_RIGHTS_GROUPS)
+        has_high_access_rights_groups = self.user_has_groups(self.HIGH_ACCESS_RIGHTS_GROUPS)
+        for user in self:
+            user.can_read_warehouse = False
+            if user.user_has_groups('stock.group_stock_multi_locations'):
```

Comment:
multi warehouse group

Reply:
and i would remove all these compute

+ i'm not aware of the groups issue in res.user? but even if they don't work, no need for these compute

Reply:
just use group_no_one if the others don't work

Reply:
so the inventory manager has no view to edit the default warehouses?

maybe we could just have a o2m between warehouses and res.partner... and thus avoid messing with all that res user crap